### PR TITLE
[stable/Sonarqube] Config map and plugin installation

### PR DESCRIPTION
**Pull request by joosthofman**
- Config map for sonar.properties
- Init container to install plugins from values
- configurable liveness and readiness probes
- custom env variables
- Small fix to static port in test pod

@rjkernick

### CODE REVIEW

File: stable/sonarqube/values.yaml [hunk 1]
```diff
@@ -42,14 +54,25 @@ persistence:
...
+
 ## Configuration values for the postgresql dependency
 ## ref: https://github.com/kubernetes/charts/blob/master/stable/postgressql/README.md
 ##
 postgresql:
   postgresUser: "sonarUser"
   postgresPassword: "sonarPass"
   postgresDatabase: "sonarDB"
+  persistence:
+    enabled: false
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml]: Not a big fan of changing that flag. What is a database without persistence?

Comment (joosthofman) [reply stable/sonarqube/values.yaml]: Fixed

**Comment by rjkernick** [comment]
Looks good. Just gotta wait for one of the org members to review.

File: stable/sonarqube/templates/config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
```
Comment (unguiculus) [inline stable/sonarqube/templates/config.yaml:4]: Add standard labels.

File: stable/sonarqube/templates/install-plugins.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
```
Comment (unguiculus) [inline stable/sonarqube/templates/install-plugins.yaml:3]: Add standard labels.

File: stable/sonarqube/values.yaml [hunk 4]
```diff
@@ -42,10 +54,19 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: ""
```
Comment (unguiculus) [inline stable/sonarqube/values.yaml]: Why this change?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/sonarqube/templates/config.yaml (hunk 2 above)
Comment (joosthofman) [reply stable/sonarqube/templates/config.yaml:4]: Thanks. Added the standard labels.

File: stable/sonarqube/templates/install-plugins.yaml (hunk 3 above)
Comment (joosthofman) [reply stable/sonarqube/templates/install-plugins.yaml:3]: Thanks. Added the standard labels.

File: stable/sonarqube/values.yaml (hunk 4 above)
Comment (joosthofman) [reply stable/sonarqube/values.yaml]: My bad. restored the storageClass: "-" value.

**Review by unguiculus — APPROVED** [review]
